### DISCUSSION TITLE
what should I see?

### DISCUSSION THREAD

**Discussion by tirelibirefe** [apache/apisix#6769] [Q&A]
Hello,
I've entered grafana url to apisix dashboard but the page is empty.
What should I see there?
![image](https://user-images.githubusercontent.com/45227339/161145248-ae6712ca-019b-40b6-916d-311ec4560957.png)

Thanks

**Comment by guitu168** [comment]
Can you see the monitoring information of Apache APISIX normally on the Grafana page? I think you can list your operation steps in detail, which is convenient for the community to troubleshoot problems.

**Reply by tirelibirefe** [reply]
no, why could I see there? I haven't loaded any dashboard for Apisix to Grafana.
...but I can see Apisix was discovered by Proemtheus.

**Comment by tirelibirefe** [comment]
I just added a dashboard to Grafana for Apisix; 
![image](https://user-images.githubusercontent.com/45227339/161232253-fa2c4bc9-96d9-4322-bfed-202376e076f5.png)

...but apisix dashboard screen is still empty.

**Reply by tirelibirefe** [reply]
sorry, how can grafana dashboard be displayed there? Grafana access needs to be authentication and authorization.

**Comment by Baoyuantop** [comment]
Hi @tirelibirefe, APISIX Dashboard supports the monitor page by referencing it in [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe). Before accessing [Grafana](https://grafana.com/), please Enable [allow_embedding=true](https://grafana.com/docs/grafana/latest/administration/configuration/#allow_embedding), which defaults to false. This causes the browser to fail to render Grafana pages properly due to security policies.

**Reply by bigbirdxixi** [reply]
i have set allow_embedding = true and set
[auth.anonymous]
enabled = true

but my page show 
"Refused to frame 'http://x.x.x.x:3000/' because it violates  the following Content Security Policy directive: "default-src 'self'". Note that 'frame-src' was not explicitly set, so 'default-src' is used as a fallback."

![image](https://user-images.githubusercontent.com/23470334/183874100-f97552e1-5943-46c6-8edc-323fb70073d0.png)

**Comment by LiberiFatali** [comment]
> i have set allow_embedding = true and set [auth.anonymous] enabled = true
> 
> but my page show "Refused to frame 'http://x.x.x.x:3000/' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'frame-src' was not explicitly set, so 'default-src' is used as a fallback."
> 
> ![image](https://user-images.githubusercontent.com/23470334/183874100-f97552e1-5943-46c6-8edc-323fb70073d0.png)

I also get this issue. Any updates?

**Reply by guitu168** [reply]
@LiberiFatali You can refer to this issue. https://github.com/apache/apisix-dashboard/issues/2586

**Reply by LiberiFatali** [reply]
Great, that works for me. Thanks!
